src/components/manage/Sidebar/ObjectBrowserBody.jsx

```diff
@@ -344,16 +343,8 @@ class ObjectBrowserBody extends Component {
    * @returns {string} Markup for the component.
    */
   render() {
-    return ReactDOM.createPortal(
-      <aside
-        role="presentation"
-        onClick={(e) => {
-          e.stopPropagation();
-        }}
-        ref={this.objectBrowser}
-        key="objectbrowsercontainerkey"
```

Comment:
I recall this was important too, to remove a warning or so... I don't see it anywhere in the refactor.